src/util/redaction.ts

```diff
@@ -21,16 +23,23 @@ export interface RedactionPolicy {
 }
 
 export const DefaultConverters: Record<DefaultConvertersNames, ValueConverter> = {
-  redact: value => '[REDACTED]',
-  contextNotAllowedImage: value => 'https://impresso-project.ch/assets/images/not-allowed.png',
+  redact: _value => '[REDACTED]',
+  contextNotAllowedImage: (value, rewriteRules) =>
+    rewriteRules
+      ? sanitizeIiifImageUrl('https://impresso-project.ch/assets/images/not-allowed.png', rewriteRules)
```

Comment:
Just pass empty rules into the function and it will figure out what to do:

```
sanitizeIiifImageUrl('https://impresso-project.ch/assets/images/not-allowed.png', rewriteRules ?? [])
```

no copy-paste this way